tests/cupy_tests/core_tests/test_raw.py

```diff
@@ -749,11 +749,18 @@ def test_context_switch_RawModule2(self):
     def test_context_switch_RawModule3(self):
         # run test_load_cubin() on another device
         # generate cubin in the temp dir and load it on device 0
-        file_path = self._generate_file('cubin')
-        mod = cupy.RawModule(path=file_path, backend=self.backend)
+
+        device0 = cupy.cuda.Device(0)
+        device1 = cupy.cuda.Device(1)
+        if device0.compute_capability != device1.compute_capability:
+            raise pytest.skip()
```

Comment:
Just a nit: could you do `self.skipTest()` to be compatible with `unittest`?

Reply:
there are some other places using `pytest.skip` instead `skipTest`, I will merge this as-is and then change all of them to obbey
https://docs-cupy.chainer.org/en/latest/contribution.html#how-to-run-tests

Reply:
Right, no worries. I think it can wait. I remember there was a discussion on which way to skip is better, but I couldn't recall where it is off top of my head.